### PR TITLE
Added a link to docs.github.com

### PR DESCRIPTION
**Pull request by wi5nia**
### Why:
Better UX

### What's being changed:
Added a links to docs.github.com

### Check off the following:
- [X] All of the tests are passing.
- [X] I have reviewed my changes in staging.
- [X] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [X] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## GitHub Docs <!-- omit in toc -->
 
-This repository contains the documentation website code and Markdown source files for docs.github.com.
+This repository contains the documentation website code and Markdown source files for [docs.github.com](docs.github.com).
```
Comment (Simran-B) [inline README.md:3]: Should use a proper URL:
```suggestion
This repository contains the documentation website code and Markdown source files for [docs.github.com](https://docs.github.com).
```

Alternatively also `<https://docs.github.com>`, which renders as: <https://docs.github.com>

**Review by Simran-B — CHANGES_REQUESTED** [review]

**Comment by janiceilene** [comment]
Thanks @wi5nia 👍 I'll update this branch and merge when the tests are green 💚

**Review by janiceilene — APPROVED** [review]

**Comment by janiceilene** [comment]
If you haven't already, you can add yourself to [the list of contributors](https://github.com/github/docs#contributors-) by creating a new comment in this PR using [these instructions](https://allcontributors.org/docs/en/bot/usage#commands). Thanks again! :sparkles:

**Comment by wi5nia** [comment]
@all-contributors please add wi5nia for doc

**Pull request merged**